community/becoming-a-maintainer.md

```diff
@@ -1,3 +1,77 @@
+
+Maintainers are also responsibile for interacting with our contributor community.
+They review PRs from community, acting as gatekeepers, shephards, and cheerleaders for our community.
+They are responsible for keeping documentation and contribution guidelines up to date, and ensuring that there are relevant open issues for potential contributors to action.
+
+It is essential that Maintainers manifest Exercism's values of empathy and kindness, while maintaining high standards for the Track and the platform.
+
+## Powers
+
+The primary way that Maintainers are able to enact their responsibilities is by having the write access to repositories. 
+Through this Maintainers control what code goes into their repositories.
+
+## Rewards
+
+The main incentives for being an Exercism Maintainer are intrinsic: working on a meaningful project; working with other smart, kind people; developing both technical and soft skills; having a meaningful out-of-work entry on your résumé.
+
+We also provide some extra benefits in:
+- A "Maintainer" badge on profiles and around the website.
+- Blog posts and interviews with maintainers.
+- Exclusive Maintainer swag
+
+## Maintainer requirements 
+
+Maintainers commit themselves to spending an average of ~4 hours per week on maintaining their Track(s).
+
+### Path to becoming a Maintainer
+
+On tracks that are actively maintained, you will be invited to become a Maintainer when you achieve 150 building and/or authoring reputation on that Track within a 6 month period. 
+
+On tracks without maintainers, we have less rigorous restrictions.
+After some small merged contributions, you can open an issue on the Track asking to become a Maintainer.
+
+### Staying a maintainer
+
+Every six months we check to see if our Maintainers are still active. 
+In those checks, anyone who has built 100 Reputation from building/authoring/maintaining track in previous 6 months is invited to continue being a maintainer.
+In general those who have done little or nothing during that window, will be removed as Maintainers.
+
+### Senior Track Maintainers
+
+Once a Maintainer has shown themselves to take active leadership within a track, and been a Maintainer on that Track for at least 6 months, we may designate them a Senior Maintainer. The general metrics we use are a Maintainer having earned >=1k rep from building/maintenance on a track, and earned > 200 rep from building/maintaining track in the last 6 months.
+
+Senior Maintainers take on added responsibility within a track, mentoring new Maintainers, and taking a stronger leadership position within a Track. 
+In decisions where there are different approaches, the opinions of Senior Maintainers hold more sway in the eyes of any arbitration by the Admin Team.
```

Comment:
```suggestion
In decisions where there are different approaches, the opinions of Senior Maintainers are weighted heavier by the Admin Team in situations that require arbitration or a decision by an uninvolved party.
```

I had never heard of "sway" as a noun before.

Reply:
I don't think it is rare in the US, but it could be in England, Australia, and other areas where English is spoken or written.  But it has a fairly  strong hold in the English language, introduced only 7 centuries ago, and not archaic in use from what I have seen.  This is one of the more familiar uses that I have seen.

Reply:
```suggestion
In decisions where there are different opinions that require arbitration by the Admin Team, the opinions of Senior Maintainers will by default hold more sway.
```

Reply:
I'm ok with keeping "hold more sway". It's also extremely common in the UK.